src/service/synchronize.toit

```diff
@@ -220,7 +220,7 @@ class SynchronizeJob extends TaskJob:
         start-watchdog_ watchdog_ max-offline
     super NAME saved-state
 
-  control --online/bool --close/bool=false -> none:
+  control --online/bool --close/bool=false --force-recovery-contact/bool=false -> none:
```

Comment:
This interface feels a bit weird, because the recovery thing makes very little sense on anything but online=true, close=false.

Reply:
changed API.